unittest/mat3d.txt

```diff
@@ -0,0 +1,90 @@
+0.010296
```

Comment:
to make these tests useful need to provide enough digits for inputs and reference outputs to be able to check tightly.